roles/kubernetes-apps/meta/main.yml

```diff
@@ -10,12 +10,6 @@ dependencies:
       - apps
       - efk
 
-  - role: kubernetes-apps/helm
```

Comment:
Please keep this code in tact. There's no reason to move it out.

Reply:
Same as cluster.yml regarding installation of helm client on one master vs all masters.

Reply:
@georgejdli I see what you were going for that's similar to the convention used by `kubernetes/client`. However, we want to avoid extra plays where we can. For helm client on all masters, perhaps it would be best to collaborate with @mirwan over #3243, and only leave the TLS options in this PR.

Thanks for the patch!

Reply:
@georgejdli Either you leave TLS options in this PR like @chadswen said or you can take my approach in your PR and I'll close mine (I wasn't aware of this PR. My apologies).

Reply:
@mirwan I'll de-scope the multi master install on my PR